warehouse/templates/pages/help.html

```diff
@@ -518,6 +520,26 @@ <h3 id="username-change">{{ username_change() }}</h3>
         <p>{% trans %}PyPI does not currently support changing a username.{% endtrans %}</p>
         <p>{% trans %}Instead, you can create a new account with the desired username, add the new account as a maintainer of all the projects your old account owns, and then delete the old account, which will have the same effect.{% endtrans %}</p>
 
+        <h3 id="openid-connect">{{ openid_connect() }}</h3>
+        <p>
+          {% trans openid="https://openid.net/connect/", apitoken="#apitoken" %}
+          PyPI users and projects can use <a href="{{ openid }}">OpenID Connect</a> to delegate
+          publishing authority for a PyPI package to a trusted third party service, eliminating
+          the need to perform manual <a href="{{ apitoken }}">API token</a> management.
+          {% endtrans %}
+        </p>
+        <p>
+          {% trans %}
+          Publishing projects with OpenID Connect is done by registering "providers,"
+          which correspond to services like GitHub Actions.
+          Providers, in turn, come in two flavors:
+          {% endtrans %}
+        </p>
+        <ul>
+          <li>{% trans %}"Normal" providers, which are registered against individual PyPI projects;{% endtrans %}</li>
+          <li>{% trans %}"Pending" providers, which are registered against authorized users.{% endtrans %}</li>
+        </ul>
```

Comment:
Flagging for review: this help text needs to be elaborated substantially, especially to distinguish between the functionality of the two provider types (and when to use them/when pending providers become normal providers).

I can do that in this PR or in a follow-up.

Reply:
I think talking about "flavors" is confusing here. We can probably just say "existing projects can add or remove OIDC providers at any time. OIDC providers can also be created in advance for projects that do not exist yet."

Reply:
SGTM! I agree it's confusing, and reflects an implementation detail more than user-surfaced behavior; I'll tweak the language.

Reply:
044738eb7